okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -125,20 +129,16 @@ public void insert(Tenant t, Handler<ExtendedAsyncResult<String>> fut) {
     String id = t.getId();
     tenants.get(id, gres -> {
       if (gres.succeeded()) {
-        fut.handle(new Failure<>(USER, messages.getMessage("10400", id)));
-      } else if (gres.getType() == NOT_FOUND) {
-        if (tenantStore == null) { // no db, just add it to shared mem
```

Comment:
The tenantStore parameter of the constructor may be null.
Can you add a null check to the constructor to fail the constructor with a NullPointerException?